partition/roles/promtail/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# TODO: move to release vector
+promtail_image_name: grafana/promtail
+promtail_image_tag: 2.7.3
```

Comment:
Would fit beneath `docker-images.third-party.partition` in the release vector.

Reply:
https://github.com/metal-stack/releases/pull/113